etc/openqa/openqa.ini

```diff
@@ -30,9 +30,16 @@
 ## Set to 0 to disable auditing backend
 # audit_enabled = 1
 
-## Set to 1 to enable profiling. Needs Mojolicious::Plugin::NYTProf
+## Set to 1 to enable profiling
+## * Needs Mojolicious::Plugin::NYTProf
+## * Profiling data will be PUBLICLY accessible under /nytprof route.
 # profiling_enabled = 0
 
+## Set to 1 to enable monitoring
+## * Needs Mojolicious::Plugin::Status
```

Comment:
shouldn't this be packaged?

Reply:
Ok, I can do that. For me `zypper in` just worked because I have `devel:languages:perl:CPAN-M` added on my system.

Reply:
So it would be then just linking packages then?

Reply:
https://build.opensuse.org/request/show/631313